storage/src/testFixtures/java/tech/pegasys/teku/storage/storageSystem/InMemoryStorageSystemBuilder.java

```diff
@@ -101,6 +101,7 @@ public StorageSystem build() {
         storageMode,
         storeConfig,
         spec,
+        Optional.empty(),
```

Comment:
at the moment no need, but in-mem could always query the EL, it would just need an async runner here